api/http/handler/deploykeys/handler.go

```diff
@@ -0,0 +1,31 @@
+package deploykeys
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/portainer/portainer"
+	httperror "github.com/portainer/portainer/http/error"
+	"github.com/portainer/portainer/http/security"
+)
+
+// Handler is the HTTP handler used to handle deploykey operations.
+type Handler struct {
+	*mux.Router
+	DeploykeyService portainer.DeploykeyService
+}
+
+// NewHandler creates a handler to manage deploykey operations.
+func NewHandler(bouncer *security.RequestBouncer) *Handler {
+	h := &Handler{
+		Router: mux.NewRouter(),
+	}
+	h.Handle("/deploykeys",
+		bouncer.AdministratorAccess(httperror.LoggerHandler(h.deploykeyCreate))).Methods(http.MethodPost)
+	h.Handle("/deploykeys",
+		bouncer.AdministratorAccess(httperror.LoggerHandler(h.deploykeyList))).Methods(http.MethodGet)
```

Comment:
I believe that the policy should be `AuthenticatedAccess` instead of `AdministratorAccess` for the `deployKeyList` operation, as any standard user account should be able to list the keys.